src/app/methods/method.jade

```diff
@@ -2,9 +2,18 @@ div(flow-init, flow-file-added="!!{png:1,gif:1,jpg:1,jpeg:1}[$file.getExtension(
   .background--image
     img(ng-hide="userBackground.image", ng-src="{{ctrl.backgroundImage}}")
     img.background-dark(ng-show="userBackground.image", flow-img="userBackground.image")
-    .form
-      .outer
+
+  .off-canvas-wrap#off-canvas(data-offcanvas)
+    .inner-wrap
+      button.tiny.button.black-button.right.right-off-canvas-toggle
+        h6 < Methods
```

Comment:
it will be really cool if we can make it so that when it is open the icon changes to `>`